cfgov/jinja2/v1/_includes/organisms/email-popup.html

```diff
@@ -0,0 +1,60 @@
+{# ==========================================================================
+
+ Email Pop-up Organism
+
+ ==========================================================================
+
+ Description:
+
+ Sidebar popup that fades in upon scrolling on the Owning a Home sublanding
+ page at /owning-a-home/.
+
+ Creates Pop-up organism:
+
+ ========================================================================== #}
+
+<div class="o-email-popup">
+    <div class="o-email-popup_header u-clearfix">
+        <div class="close">
+            <a>Close <span class="cf-icon cf-icon-delete-round"></span></a>
+        </div>
+    </div>
+    <div class="o-email-popup_body">
+        <div class="o-email-signup">
+            <h2>Buying a home?</h2>
+            <form id="{{ ('o-email-signup_' ~ range(1, 100) | random) }}"
+                  class="o-form o-form__email-signup u-clearfix"
+                  action="/subscriptions/new/"
+                  method="POST"
+                  enctype="application/x-www-form-urlencoded"
+                 novalidate>
+                <p>Sign up for email tips and info to help you through the process.</p>
+                <div class="m-form-field-with-button" data-qa-hook="formfield-with-button">
+                    <div class="form-group">
+                        <input id="form_18"
+                               type="email"
+                               placeholder="Enter your email address"
+                               name="email"
+                               class="m-form-field-with-button_field"
```

Comment:
Does the email address input need the `a-text-input` class? 

<img width="585" alt="screen shot 2017-10-18 at 11 15 47 am" src="https://user-images.githubusercontent.com/778171/31735962-3bba5754-b3f8-11e7-9dc7-23994eb04319.png">

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3484/commits/5fd38ec2dd69e636764b679629644d599226fbdc